### PR TITLE
Build Docker image

### PR DESCRIPTION
**Pull request by shumkov**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
+
+branches:
+  only:
+    - master
+    # Allow builds for version tags (e.g. v1.0.0)
+    - /^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/
+
+before_install:
+  # Login to NPM registry
+  - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > .npmrc
```
Comment (nmarley) [inline .travis.yml:26]: This shouldn't be necessary

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
+FROM node:8-alpine
+
+RUN apk add --update --no-cache \
+                            git \
+                            libzmq \
+                            zeromq-dev \
+                            python \
+                            make \
+                            g++ \
```
Comment (nmarley) [inline Dockerfile]: This is kinda ugly, sure you don't want to put these all on one line? The final probably doesn't need the `\`, but not sure how this affects w/an empty line after.

File: .travis.yml [hunk 3]
```diff
@@ -1,10 +1,35 @@
...
 install:
-  - npm install
+  - npm ci
+
+deploy:
+  - provider: script
+    skip_cleanup: true
+    script: bash scripts/travis-deploy.sh
+    on:
+      repo: dashevo/insight-ui
```
Comment (nmarley) [inline .travis.yml]: Is this intentionally the `-ui` or should it be `-api` here instead?

File: .travis.yml [hunk 4]
```diff
@@ -1,10 +1,35 @@
 sudo: false
 dist: trusty
 language: node_js
+
 node_js:
```
Comment (nmarley) [inline .travis.yml:5]: Why removing these? They should be running for all supported Node.js versions, right?

**Review by nmarley — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (shumkov) [reply .travis.yml:26]: We need this for publishing npm package.

File: Dockerfile (hunk 2 above)
Comment (shumkov) [reply Dockerfile]: I was found it more readable and got used to writing like that. Also, it easier to include/exclude packets.

File: .travis.yml (hunk 3 above)
Comment (shumkov) [reply .travis.yml]: Good point, sorry.

File: .travis.yml (hunk 4 above)
Comment (shumkov) [reply .travis.yml:5]: Oh, you right. My bad.

File: Dockerfile (hunk 2 above)
Comment (nmarley) [reply Dockerfile]: ¯\\\_(ツ)\_/¯ Ok.

**Review by nmarley — APPROVED** [review]
utACK

**Pull request merged**